### ISSUE TITLE
How to add tailwind style to a button?

### ISSUE THREAD

**Issue by zfusx** [reflex-dev/reflex#2366]
**Describe the bug**
Add the tailwind style to button , no effect on that button

`rx.button("Click me", class_name="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded")`

![Screen Shot 2024-01-08 at 7 32 33 AM](https://github.com/reflex-dev/reflex/assets/155436867/7d5ef033-810e-49d4-a2bc-a643c749eabe)

in Chrome we can see the button style is:
`<button type="button" class="chakra-button bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded css-wqpdoh">Click me</button>`

if we remove the `css-wqpdoh` from the class,

the button is tailwind styled:
![Screen Shot 2024-01-08 at 7 33 21 AM](https://github.com/reflex-dev/reflex/assets/155436867/18ed6a2f-ceb3-4e7b-97d1-75b673b6ade4)

what's the problem with the button component?

where does the `css-wqpdoh` and `chakra-button` class comes from? any idea to block it, 
It's easy to get pure tailwind class from designer, otherwise it would be headache to convert tailwind css to other style ,huge repeat work!

**Specifics (please complete the following information):**
 - Python Version: 3.10
 - Reflex Version: reflex==0.3.7
 - OS: Mac OS
 - Browser (Optional): Chrome

**Comment by cllatser** [comment]
Hello, there is an issue with Chakra and Tailwind components, they don't play well together. Reflex is switching Chakra components to Radix components.
My advice if you want to style with Tailwind is to use native HTML tags. In your case, it would be `rx.el.button()`.
Best regards.

**Comment by zfusx** [comment]
> rx.el.button()

![Screen Shot 2024-01-08 at 5 42 56 PM](https://github.com/reflex-dev/reflex/assets/155436867/dc598bc8-b3fb-4888-983f-a53d3d52c664)
you are the Man !

**Comment by cllatser** [comment]
> If you feel comfortable it is the html tags. You can do `from reflex import el`  and this, it is not necessary to put `rx.` in front of. So it would be `el.button()`, `el.div()`, `el.input()` 😀

**Comment by Lymah123** [comment]
Hi @zfusx has anyone started working on the issue?

**Comment by zfusx** [comment]
> Hi @zfusx has anyone started working on the issue?

I don't know we just put into testing, normally designer will give figma with css in tailwind, if we can do all in python, that will be really great, nobody like js and its framework 's Spaghetti code

**Comment by Alek99** [comment]
Yes I would also advise using the components soon our core components will be able to support tailwind but probably won't be ready for a couple weeks
